From: Tommy Webb <tommy@calyxinstitute.org>
Date: Tue, 27 May 2025 16:25:29 +0000
Subject: Update text for default browser-based autofill

Chrome uses Google and Google Password Manager for autofill,
but we still use local password storage. Update the text of the
default choice to reflect this.

Note that "Chrome" is still used in this new text, as it is already
replaced automatically by other methods.

Change-Id: Ia3a62e60e5f065d935151c39a8a9518c2155dc67
---
 .../layout/radio_button_group_third_party_preference.xml    | 4 ++--
 components/autofill_strings.grdp                            | 6 ++++++
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/chrome/browser/autofill/android/java/res/layout/radio_button_group_third_party_preference.xml b/chrome/browser/autofill/android/java/res/layout/radio_button_group_third_party_preference.xml
--- a/chrome/browser/autofill/android/java/res/layout/radio_button_group_third_party_preference.xml
+++ b/chrome/browser/autofill/android/java/res/layout/radio_button_group_third_party_preference.xml
@@ -20,8 +20,8 @@ found in the LICENSE file.
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
           android:checked="true"
-          app:primaryText="@string/autofill_third_party_filling_default"
-          app:descriptionText="@string/autofill_third_party_filling_default_description"/>
+          app:primaryText="@string/autofill_third_party_filling_default_calyx"
+          app:descriptionText="@string/autofill_third_party_filling_default_description_calyx"/>
 
         <org.chromium.components.browser_ui.widget.RadioButtonWithDescription
           android:id="@+id/autofill_third_party_filling_opt_in"
diff --git a/components/autofill_strings.grdp b/components/autofill_strings.grdp
--- a/components/autofill_strings.grdp
+++ b/components/autofill_strings.grdp
@@ -362,12 +362,18 @@
     </message>
   </if>
   <if expr="is_android">
+    <message name="IDS_AUTOFILL_THIRD_PARTY_FILLING_DEFAULT_CALYX" desc="Title of the option to use the default Chrome autofill experience." formatter_data="android_java">
+      Autofill with Chrome
+    </message>
     <message name="IDS_AUTOFILL_THIRD_PARTY_FILLING_DEFAULT" desc="Title of the option to use the default Chrome autofill experience." formatter_data="android_java">
       Autofill with Google
     </message>
     <message name="IDS_AUTOFILL_THIRD_PARTY_FILLING_OPT_IN" desc="Title of the option to use third-party autofill in Chrome." formatter_data="android_java">
       Autofill using another service
     </message>
+    <message name="IDS_AUTOFILL_THIRD_PARTY_FILLING_DEFAULT_DESCRIPTION_CALYX" desc="Description shown on the option to use the default Chrome autofill experience." formatter_data="android_java">
+      Use Chrome Autofill to save and fill passwords, payment methods, and addresses
+    </message>
     <message name="IDS_AUTOFILL_THIRD_PARTY_FILLING_DEFAULT_DESCRIPTION" desc="Description shown on the option to use the default Chrome autofill experience." formatter_data="android_java">
       Use Google Password Manager and Chrome Autofill to save and fill passwords, payment methods, and addresses
     </message>
-- 

